### PR TITLE
Readme edits and additions

### PR DESCRIPTION
**Pull request by riddlez666**
Edited the readme to reflect our git and also added the json files for pool setup as examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +311,7 @@ This pool is tested to work with [Ethcore's Parity](https://github.com/ethcore/p
 ### Credits
 
 Made by sammy007. Licensed under GPLv3.
+Modified by Ellaism
```
Comment (ellaismer) [inline README.md]: I'm going to change this to `Modified by The Ellaism Project` for consistency.

**Comment by ellaismer** [comment]
The config files looks redundant though... Not everyone needs all configs. For really small pools, it is indeed possible to just use one config to run the whole process. So I'd say let's just keep one example config for ease of update. I'm going to cherry pick the README changes to master.

**Comment by ellaismer** [comment]
Cherry-picked at https://github.com/ellaism/open-ethereum-pool/commit/e5de333e4297f3bf99b5d5aba100cd6473759b44

**Comment by riddlez666** [comment]
I'm going to close and do a new PR with a proper folder and layout.

**Pull request closed without merging**